sdk/storage/inc/blobs/internal/blob_client_options.hpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+#include <limits>
+#include <utility>
+
+#include "protocol/blob_rest_client.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  struct BlobClientOptions
+  {
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+  };
+
+  struct BlockBlobClientOptions : public BlobClientOptions
+  {
+
+  };
+
+  struct GetBlobPropertiesOptions
+  {
+    Azure::Core::Context context;
+  };
+
+  struct SetBlobHttpHeadersOptions
+  {
+    Azure::Core::Context context;
```

Comment:
>context [](start = 25, length = 7)

Uppercase `C` for `Context``?

Reply:
Applies to other context as well.

---
In reply to: [433601567](https://github.com/Azure/azure-sdk-for-cpp/pull/144#discussion_r433601567) [](ancestors = 433601567)

Reply:
fixed